src/tests/end-to-end/common/force-test-failure.ts

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 export function forceTestFailure(errorMessage: string): void {
-    process.emit('uncaughtException', new Error(errorMessage));
+    expect(errorMessage).toBeNull();
```

Comment:
process.emit was not failing the test.
considered passing the done.fail method from it/test callback function. this is tricky. it would make us pass the done.fail everywhere. Also, we memorize background page in browser whose scope may be test class instead of test itself.

Reply:
here is  a sample failing test - [https://dev.azure.com/ms/accessibility-insights-web/_build/results?buildId=7758&view=ms.vss-test-web.build-test-results-tab](url) 
![image](https://user-images.githubusercontent.com/29855291/55025827-ee2fc780-4fbe-11e9-918d-44c06e77fcc1.png)